.github/ISSUE_TEMPLATE/PPR_TEMPLATE.md

```diff
@@ -0,0 +1,206 @@
+## Production Readiness Review (PRR) 
+
+| Feature       | `[TICKET_NUMBER]: FeatureName`         |
+| ------------- | -------------------------------------- |
+| Authors       | `author_name`                          |
+| GitHub Handle | `github_handle`                        |
+| Start Date    | `YYYY-MM-DD`                           |
+| End Date      | `YYYY-MM-DD`                           |
+| Status        | `Not Started` / `In Progress` / `Done` |
+
+### References
+
+| Category          | Links                                  |
+| ----------------- | -------------------------------------- |
+| GitHub repository | `org/repository`                       |
+| Requirements      | `link to documents`                    |
+### Approval Documents
+
+- [ ] PIA: `<location link>`
+- [ ] STRA: `<location link>`
+- [ ] FRCR: `<location link>`
+- [ ] PenTest: `<location link>`
+
+### Reviewers
+
+Once review has been completed, corresponding reviewer will check the box next to their name when the review is completed. If there are any subordinate tasks, please link in the issue
+
+#### Required
+
+- [ ] SRE Team: `<reviewer name>`
+- [ ] Infosec Team: `<reviewer name>`
+- [ ] Product Owner: `<reviewer name>`
+#### Optional
+
+- [ ] Database Team: `<reviewer name>`
+- [ ] Development Team: `<reviewer name>`
+
+### Architectrure
+
+- [ ] Is there are high level summary of the product feature documented along with use-cases ? 
+- [ ] Are there any architecture diagrams which shows the feature and how it interact with other services / features. Please add following: dependencies ( internal / external , ports, encryption, protocols, ACLs, security policies etc )
+- [ ] Are there interaction or sequence diagrams of critical components ?
+- [ ] Is there a SBOM ?
+- [ ] Do all diagrams have a textual / versioned source ? (eg Visio Image with accompanying VSDX file, formats like mermaid in markdown files do not need a separate file)
+
+### Operational / Scaling risks
+
+- [ ] What are the potential scaling issues / design tradeoffs that were made
+- [ ] List all the dependencies and mention if the dependency is `soft/hard` and how service might be impacted by failure of that dependency ?
+- [ ] What is the blast radius if this feature fails ?
+- [ ] Are there any SPOF in this feature design and if yes, how are we going to handle that ?
+- [ ] How easily can we scale the service ?
+- [ ] How are we going to scale the service ( horizontal or vertical ) ?
+
+### SLA / SLO / SLI
+
+- [ ] Is there a SLA defined for this feature / service ? If yes, then list out all the associated SLI and map to each SLA ?
+- [ ] List out SLOs ?
+- [ ] Map SLI each SLO ?
+- [ ] How are the SLIs measured ?
+
+#### CI/CD
```

Comment:
do we need a generic post deployment verification checklist item?

Reply:
maybe, but that would be specific to each service. we could add it as a suggestion though

Reply:
yeah, i guess it could be just an item on this list like "Is there a post deployment verification checklist?" and the actual checklist will differ per service